[ADD] new doc for release report (#1272)
mercadopago/devsite-docs: guides/manage-account/reports/available-money/api.en.md

# Generating through API

Generate the Released money report manually as many times as you want or schedule it according to the desired frequency through our API.

> WARNING
>
> The Available Balance report will be disabled from March 1st, 2022.
>

> > Use the [Release report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to reconcile the transactions that affect the balance available in your account, including your bank withdrawals.

Reviewer: ```suggestion
> You can use the [Release report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to reconcile the transactions that affect the balance available in your account, including your bank withdrawals.
```

## Set up your reports

> Execute the curl you need to create, review and update your reports.

Reviewer: ```suggestion
You can configure your reports according to your preferences. Up next, you'll find the API calls you can make to create, consult and update your reports.
```

> ### Create configuration

Reviewer: ```suggestion
### Create the configuration
```

> WARNING
>
> The Available Balance report will be disabled from next March 1st, 2022. 
>

> > Use the [Releases report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to reconcile the transactions that affect the balance available in your account, including your bank withdrawals.

Reviewer: ```suggestion
> Use the [Releases report](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/reports/released-money/introduction) to the transactions that affect the balance available in your account, including your bank withdrawals.
```

}
```

> ### Check configuration 

Reviewer: ```suggestion
### Check the configuration 
```
